hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -1860,11 +1856,14 @@ public boolean balance(boolean force) throws IOException {
         //rpCount records balance plans processed, does not care if a plan succeeds
         rpCount++;
 
-        balanceThrottling(balanceStartTime + rpCount * balanceInterval, maxRegionsInTransition,
+        if (this.maxBlancingTime > 0) {
```

Comment:
ok so by disabling throttling what we mean is providing negative int value for config `hbase.balancer.max.balancing` in master site config

Reply:
Yes.

Reply:
So you will change this maxBlancingTime dynamically at HM config?  
If permanently for the cluster the throttling needs disabled, hbase.master.balancer.maxRitPercent can be 100% right (which is the def value).  What am missing here? I did not read this part of code fully though

Reply:
No this do not change maxBlancingTime dynamically, It just skip the balanceThrottling when maxBlancingTime<=0.